test/integration/etcd_storage_path_test.go

```diff
@@ -677,20 +729,73 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	// --
 }
 
-// Be very careful when whitelisting an object as ephemeral.
+// Be very careful when whitelisting a kind as unexposed.
 // Doing so removes the safety we gain from this test by skipping that object.
-var ephemeralWhiteList = createEphemeralWhiteList(
-	// github.com/openshift/origin/pkg/authorization/apis/authorization/v1
+// These types aren't exposed via the API
```

Comment:
what goes here and not to the ephemeral list?

Reply:
> what goes here and not to the ephemeral list?

Things like ResourceAccessReviewResponse, which isn't ephemeral, it's just never exposed via the API

Reply:
@stevekuznetsov ptal

Reply:
Seems reasonable enough, though I wish there was a nicer way to know if a resource should be skipped.

I find all the TODOs in that file far more concerning 😄